Webapp/sources/blueprints/reaction_table/routes.py

```diff
@@ -28,15 +30,15 @@ def process():
 
     # get user workbook
     demo = request.args.get("demo")
+    tutorial = request.args.get("tutorial")
     reaction = None
     workbook = None
-    if demo != "demo":
+    if demo != "demo" and tutorial == "tutorial":
```

Comment:
I'm not following the logic here. We only assign workgroup/book/reaction if in tutorial mode?

Reply:
good spot, it should be: tutorial != "tutorial"

Reply:
> good spot, it should be: tutorial != "tutorial"

this should be fixed now, needed to send the tutorial status in the URL from the javascript and change this to tutirla =="yes" (thanks for the help)